controllers/reconcilers/console_plugin/plugin_installation_reconciler.go

```diff
@@ -144,47 +154,50 @@ func (r *Reconciler) reconcileService(ctx context.Context) (v1alpha1.PlatformsIn
 			},
 		}
 		service.Spec.Selector = map[string]string{
-			"app": DBaaSDynamicPluginName,
+			"app": r.pluginName,
 		}
 		service.Spec.Type = v1.ServiceTypeClusterIP
 		service.Spec.SessionAffinity = v1.ServiceAffinityNone
 		return nil
 	})
 
 	if err != nil {
+		if errors.IsConflict(err) {
+			return v1alpha1.ResultInProgress, nil
+		}
 		return v1alpha1.ResultFailed, err
 	}
 	return v1alpha1.ResultSuccess, nil
 }
 
 func (r *Reconciler) reconcileDeployment(ctx context.Context) (v1alpha1.PlatformsInstlnStatus, error) {
-	deployment := GetDBaaSDynamicPluginDeployment()
+	deployment := r.getDeployment()
 	_, err := controllerutil.CreateOrUpdate(ctx, r.client, deployment, func() error {
 		deployment.Labels = map[string]string{
-			"app":                                DBaaSDynamicPluginName,
-			"app.kubernetes.io/component":        DBaaSDynamicPluginName,
-			"app.kubernetes.io/instance":         DBaaSDynamicPluginName,
-			"app.kubernetes.io/part-of":          DBaaSDynamicPluginName,
-			"app.openshift.io/runtime-namespace": DBaaSDynamicPluginName,
+			"app":                                r.pluginName,
+			"app.kubernetes.io/component":        r.pluginName,
+			"app.kubernetes.io/instance":         r.pluginName,
+			"app.kubernetes.io/part-of":          r.pluginName,
+			"app.openshift.io/runtime-namespace": r.pluginName,
```

Comment:
can we add a separate variable for the namespace, I know that currently, both are the same, but just in case if we need a different namespace for installation? WDYT?

Reply:
added the namespace variable.